src/main/java/com/example/LooseCouplingDemo/service_implementation/SMSMessagesServiceImplInterface.java

```diff
@@ -0,0 +1,42 @@
+package com.example.LooseCouplingDemo.service_implementation;
+
+import com.example.LooseCouplingDemo.dto.AddMessagesDTO;
+import com.example.LooseCouplingDemo.mapper.MessagesMapper;
+import com.example.LooseCouplingDemo.model.Messages;
+import com.example.LooseCouplingDemo.repository.MessagesRepository;
+import com.example.LooseCouplingDemo.service_Inteface.MessageServiceInterface;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service("sms")
+public class SMSMessagesServiceImplInterface implements MessageServiceInterface {
```

Comment:
If this class is implementing an interface why does it have `Interface` as a suffix?

Reply:
Done